logeco_custom/hooks.py

```diff
@@ -0,0 +1,69 @@
+# Inspiration from OCA/project/project_task_code
+from odoo import api, SUPERUSER_ID
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
```

Comment:
What is this _sorcery_? Can you explain on monday? :-)

Reply:
Pre init hooks, see comments for explanation. Live explanation available upon request. This is currently sub-optimal and require a manual db intervention, so I'd like to hear your thoughts!